doc/sphinxext/numpy_ext/docscrape_sphinx.py

```diff
@@ -27,7 +27,6 @@ def _str_indent(self, doc, indent=4):
         return out
 
     def _str_signature(self):
-        return ['']
```

Comment:
I don't understand this change. If the ``_str_signature`` is ever called, this is the line that would be reached, right?
Also, this is part of the numpy sphinx ext and any changes to it should go to their repo.